examples/chatmemberbot.py

```diff
@@ -136,11 +139,29 @@ async def greet_chat_members(update: Update, context: ContextTypes.DEFAULT_TYPE)
         )
 
 
+async def start(update: Update, context: ContextTypes.DEFAULT_TYPE) -> None:
+    """Greets the user and records that they started a chat with the bot
+    if it's a private chat. Since no automatic update is issued when a user starts a private chat
+    with the bot, we have to track it explicitly here, when handling the /start command.
+    """
+    cause_name = update.effective_user.full_name
+    chat = update.effective_chat
+    if chat.type == Chat.PRIVATE:
+        logger.info("%s started a private chat with the bot", cause_name)
+        context.bot_data.setdefault("user_ids", set()).add(chat.id)
+
+    await update.effective_message.reply_text(
+        rf"Welcome {cause_name}. Use /show_chats command to see what chats I'm in."
```

Comment:
```suggestion
        rf"Welcome {cause_name}. Use /show_chats to see what chats I'm in."
```
why the raw string though?

Reply:
Oh, I just confused the slashes at first, thought I had to put a backslash, and it required a raw string. Then remembered I was wrong but forgot to remove the raw string :)
Done